src/index.html

```diff
@@ -1,13 +1,312 @@
+            <button
+              type="submit"
+              class="button button--explore"
+            >
+              Explore
+            </button>
+          </div>
+        </div>
+      </section>
+
+      <section class="contact" id="contact">
+        <div class="contact__content">
+          <h2 class="contact__title">Contact us</h2>
+          <div class="field">
+            <form
+              action="#"
+              method="post"
+              onsubmit="this.reset(); return false;"
+            >
+              <div class="input input--name">
+                <input
+                  type="text"
+                  name="name"
+                  id="name"
+                  placeholder="Name"
+                >
+              </div>
+              <div class="input input--email">
+                <input
+                  type="email"
+                  name="email"
+                  id="email"
+                  placeholder="Email"
+                >
+              </div>
+              <div class="textarea">
+                <textarea
+                  name="comments"
+                  id="textarea"
+                  cols="30"
+                  rows="12"
+                  placeholder="Massage"
+                ></textarea>
+              </div>
+              <div class="button-send">
+                <button
+                  class="button button--send"
+                  type="submit"
+                >
+                  Send
+                </button>
+              </div>
+            </form>
+          </div>
+
+          <address class="contact__bottom">
+            <p class="contact__bottom-title contact__bottom-title--phone">Phone</p>
+            <a
+              href="tel:+1 234 5555-55-55"
+              class="contact__bottom-information contact__bottom-information--phone-number"
+            >
+              +1 234 5555-55-55
+            </a>
+            <p class="contact__bottom-title contact__bottom-title--email">Email</p>
+            <a
+              href="mailto:hello@miami.com"
+              class="contact__bottom-information contact__bottom-information--email-address"
+            >
+              hello@miami.com
+            </a>
+            <p class="contact__bottom-title contact__bottom-title--adres">Address</p>
+            <a
+              href="https://www.google.com.ua/maps/place/400+N+1st+Ave+STE+700,+Minneapolis,+MN+55401,+%D0%A1%D0%A8%D0%90/@44.9793807,-93.2790691,15.5z/data=!4m6!3m5!1s0x52b3328fdf233887:0x3348e5113ed2472d!8m2!3d44.9813258!4d-93.2731311!16s%2Fg%2F11nyn_tfyt?hl=ru&entry=ttu"
+              class="contact__bottom-information contact__bottom-information--map-address"
+              target="_blank"
+            >
+              400 first ave.<br>
+              suite 700<br>
+              Minneapolis, MN 55401
+            </a>
+          </address>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer"></footer>
```

Comment:
```suggestion
    <footer class="footer" />
```
just FYI - you may use self-closing tags in such cases